parser/mysql/const.go

```diff
@@ -155,6 +155,9 @@ const (
 	ClientPluginAuth
 	ClientConnectAtts
 	ClientPluginAuthLenencClientData
+	ClientHandleExpiredPasswords
+	ClientSessionTrack
+	ClientDeprecateEOF
```

Comment:
Maybe address morgo's comments in another PR, https://github.com/pingcap/tidb/pull/32119/files#r800834737

Reply:
Done in 970508b5ec2612910b65639bda6521d821fd0072